controllers/observability_controller.go

```diff
@@ -225,21 +224,21 @@ func (r *ObservabilityReconciler) InitializeOperand(mgr ctrl.Manager) error {
 		return err
 	}
 
-	storageClassExists := false
-	storageClass := &storage.StorageClass{}
-	selector := client.ObjectKey{
-		Name: storageClassGp2,
+	runningOnCluster := false
```

Comment:
Even if it is running locally, it is still running on a cluster. Maybe `runningOnLocalCluster` or `runningLocally` would be a more apt name for this variable?

Reply:
I see what you mean, how about something like `runningOnCloud` or `runningOnProvider`?

Reply:
`runningOnCloud` would do.